controls/package_spec.rb

```diff
@@ -101,3 +101,74 @@
     its('disk_error_action') { should cmp 'SUSPEND' }
   end
 end
+
+
+control 'package-09' do
+  impact 1.0
+  title "Install packages suggested by Lynis"
```

Comment:
@artem-sidorenko We might remove them. However, I just kept them as Lynis-hardening-score was dependent on it and I had goals to have way higher lynis-hardening index :)